File: dataset/python2java/python/PANGRAM_CHECKING.py
def f_gold ( s ) :
    List = [ ]
    for i in range ( 26 ) :
        List.append ( False )
    for c in s :
        if 'A' <= c and c <= 'Z':
            List [ ord ( c ) - ord ( 'A' ) ] = True
        elif 'a' <= c and c <= 'z':
            List [ ord ( c ) - ord ( 'a' ) ] = True
    for ch in List :
        if ch == False :
            return False
    return True

File: result/translate/transcoder/python2java/java/PANGRAM_CHECKING.java
public static boolean f_gold ( String s ) {
  boolean [ ] List = new boolean [ 26 ] ;
  for ( int i = 0 ;
  i < List . length ;
  i ++ ) List [ i ] = false ;
  for ( int i = 0 ;
  i < s . length ( ) ;
  i ++ ) {
    char c = s . charAt ( i ) ;
    if ( 'A' <= c && c <= 'Z' ) List [ ( int ) ( c - 'A' ) ] = true ;
    else if ( 'a' <= c && c <= 'z' ) List [ ( int ) ( c - 'a' ) ] = true ;
  }
  for ( boolean ch : List ) if ( ch == false ) return false ;
  return true ;
}